eclipse/tests/org.switchyard.tools.ui.tests/src/org/switchyard/tools/ui/tests/CreateSwitchYardProjectTest.java

```diff
@@ -353,4 +358,14 @@ protected void execute(IProgressMonitor monitor) throws CoreException {
         
     }
     
+    public void testVersionsListHasDefaultAndPreviousVersions() throws Exception {    
```

Comment:
optional: might worth adding a second test to ensure that they are not added twice